src/test/java/com/amazon/opendistroforelasticsearch/security/SlowIntegrationTests.java

```diff
@@ -75,7 +75,9 @@ public void testNodeClientAllowedWithServerCertificate() throws Exception {
                 .put("node.data", false)
                 .put("node.master", false)
                 .put("node.ingest", false)
-                .put("path.home", "/tmp")
+                .put("path.data", "./target/data/" + clusterInfo.clustername + "/cert/data")
```

Comment:
The changes look good, but I was wandering if using prefix "./target" is a good idea ? 

I saw that this folder is used for the output jars during package build.    
Maybe instead we should create some temporary folders or use different prefix "test-target" (or anything that contains test) .   
Do we have other tests that also use "./target" ?

Reply:
"target" is the default location that maven uses for builds and temporary files including compiler output, unit test (surefire) reports, jars, and more. As jars are created from "target/classes" or "target/test-classes", there will be no conflict and no unnecessary files will be included into the target jar files.

The goal of the change is to use "mvn clean" to delete any leftover directories/files that were not cleaned/deleted during the tests.